game/static/game/css/board.css

```
@@ -1416,18 +1938,784 @@ body {
+@media (prefers-reduced-motion: reduce) {
+    .thinking-dots span {
+        animation: none;
+        opacity: 0.7;
+    }
+}
+
+.theme-toggle-btn {
+    width: 38px;
+    height: 38px;
+
+    border-radius: 50%;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)